.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+
+before_install:
+    - sudo apt-add-repository -y ppa:ubuntu-toolchain-r/test
+    - sudo apt-add-repository -y ppa:beineri/opt-qt542
+    - sudo apt-get -qq update
+    - sudo apt-get -qq install g++-4.8 libc6-i386 qt54tools qt54svg qt54webkit qt54serialport

```

Comment:
Ahh, I'll have to go read more about how a travis build works, I only read up on it briefly